src/MagnetClasses.cc

```diff
@@ -143,6 +143,10 @@ MagnetBase* MagnetBase::MagnetFactory(G4String inputString, DetectorConstruction
         theMagnet = new MagnetTARGETR             (magnetPos, doRelPos, magnetLength, magnetGradient,
                                                    keyValPairs, detCon, magnetName);
     }
+    else if(magnetType == "PBW") {
```

Comment:
Specialized magnet type, please moved to the bottom of the list, after SCHIELDEDSCINTILLATOR